spec/unit/recipes/source_spec.rb

```diff
@@ -0,0 +1,57 @@
+require 'spec_helper'
+
+describe 'prometheus::source' do
+  let(:chef_run) do
+    ChefSpec::SoloRunner.new(file_cache_path: '/var/chef/cache') do |node|
+      node.set['root_group'] = 'root'
+      node.set['prometheus']['source']['version'] = '0.11.1'
+    end.converge(described_recipe)
+  end
+
+  it 'includes build-essential' do
+    expect(chef_run).to include_recipe('build-essential::default')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.